packages/docs/docs/advanced/canvas-operations.mdx

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_position: 5
+slug: /canvas-operations
+---
+
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+
+# Canvas Operations
+
+Because Motion Canvas is build ontop of the Browser's 2D Rendering Context, one
+can make use several canvas operations that are provided by the Browser.
+
+## Filters
+
+<AnimationPlayer name="canvas-operations-filters" />
+
+Every Component accepts a list of filters. You can find a list of all filters
+the Browser can understand on
+[MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/filter).
+
+All filters available in Motion Canvas can be found under
+`@motion-canvas/2d/lib/partials`
+([API Documentation](/api/2d/partials#Functions)). Note that some filters, like
+`opacity`, can be set directly on the Node, and as are not exported from
+`partials`.
+
+`filters` accepts an array. This means that you can stack multiple filters onto
+a single Node. The order in which you add them does matter, as can be seen
+below:
+
+<AnimationPlayer name="canvas-operations-filters-stacking" />
+
+You can see that for example the green trail turns yellow on the right side
+while remaining green on the left.
+
+## Masking and Composite Operations
+
+Composite Operations define how two layers are stacked ontop of each other. This
+way it is possible to define Masks. MDN has a
+[great visualisation of all Composite Operations](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalCompositeOperation#operations).
+
+If you are familiar with Adobe After Effects, you might know about
+[Track Mattes](https://helpx.adobe.com/after-effects/using/track-mattes-and-traveling-mattes.html).
+Track Mattes allow you to define a Background "Color" Layer, and a Foreground
+"Stencil" Layer. The Alpha Matte Layer for example will multiply the
+background's color by the stencil's alpha value.
```

Comment:
What about those who aren't? wouldn't a more generalised description of masks be better? (although keep the link I think it may be helpful)

Don't get me wrong, this is a great analogy but wouldn't revolve my explanation around it. Relying on some prior knowledge from other software may be unintuitive, especially seeing as adobe is platform specific and quite pricey.

The best i can come up with is cutting shapes out of card and layering the cards but I doubt its the best.

Reply:
I would argue that L44-L46 define what a Track Matte is for people that never touched AfterFX. I could go into more detail though if you think that this is not the case.

Reply:
I'd like to avoid mentioning AE throughout the docs. 
This can give people a false idea that Motion Canvas is an alternative to it.
I also don't want to risk any sort of legal repercussions.

I agree with @nick-1666 that masks are a more general concept used in graphic software